e2e/cephfs.go

```diff
@@ -298,7 +306,12 @@ var _ = Describe("cephfs", func() {
 
 			By("create PVC in storageClass with volumeNamePrefix", func() {
 				volumeNamePrefix := "foo-bar-"
-				err := createCephfsStorageClass(f.ClientSet, f, false, map[string]string{"volumeNamePrefix": volumeNamePrefix})
+				err := createCephfsStorageClass(
+					f.ClientSet,
+					f,
+					false,
+					map[string]string{"volumeNamePrefix": volumeNamePrefix},
```

Comment:
The trailing comma suggests that additional options are possible, but I do not think that `createCephfsStorageClass()` supports that. Is the comma really required?

Reply:
sure, removed the `traliing comma` here,